src/ui/menu/MotionData.qml

```diff
@@ -178,6 +184,43 @@ MenuItem {
             text: qsTr("Pitch is camera angle up/down when using FPV blackbox data");
         }*/
     }
+    CheckBoxWithContent {
+        id: gyrobias;
+        text: qsTr("Gyro bias");
+        onCheckedChanged: update_bias();
+        function update_bias() {
+            controller.set_imu_bias(gyrobias.checked? bx.value : 0, gyrobias.checked? by.value : 0, gyrobias.checked? bz.value : 0)

```

Comment:
`Qt.callLater(controller.set_imu_bias, gyrobias.checked? bx.value : 0, gyrobias.checked? by.value : 0, gyrobias.checked? bz.value : 0);`
this will prevent it from calling multiple times when X,Y and Z are set together